data/tlg0062/tlg054/tlg0062.tlg054.perseus-grc3.xml

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="grc">Περὶ τῶν διψάδων</title>
```

Comment:
As with above we usually just have the Greek language title in the Greek language files.